packages/editor/src/components/parts/mail/MailHeaderPart.test.tsx

```diff
@@ -82,6 +82,6 @@ describe('MailHeaderPart', () => {
     expect(data.config.headers.bcc).toEqual('');
     expect(data.config.headers.to).toEqual('');
     expect(data.config.failIfMissingAttachments).toBeFalsy();
-    expect(data.config.exceptionHandler).toEqual('ivy:error:email');
+    expect(data.config.exceptionHandler).toEqual('');
```

Comment:
Because both db query and email do use the exceptionHandler parameter, but both with another default value (ivy:error:email for mail, and ivy:error:database for db). 
Currently the test default value was on ivy:error:email, but I now set it to an empty string.
As this has only some testing effects I thought I makes no difference, but if you like I can add the correct default via parameter